R/main_script.R

```diff
@@ -47,21 +52,21 @@ call_api_once <- function(URL, path = NULL, query, ...) {
 #' @importFrom dplyr %>%
 call_api_multiple_times <- function(iterator, URL, path, query, resultVector, use_parallel = TRUE) {
   message("Making ", length(iterator), " calls.")
-
+  
```

Comment:
What happened to all the indentations?

Reply:
I am not sure. It looks like my RStudio did it automatically, but I don't see any settings for it